agent/containers/images/image-test-script.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+# A demonstration of how to use pbench-tool-meister-start with an existing
+# Redis server, and one or more remote
+
+read -p "Specify distribution to test: " distrovar
+DISTRO=$distrovar
+
+read -p "Specify image tag to test: " tagvar
+TAG=$tagvar
+
+read -p "Specify redis host: " redishost
+read -p "Specify redis port: " redisport
+export REDIS_HOST=$redishost
+export REDIS_PORT=$redisport
+export PBENCH_REDIS_SERVER="${REDIS_HOST}:${REDIS_PORT}"
+
+function wait_keypress {
+    echo "Press any key to continue"
+    while [[ true ]]; do
+        read -t ${1} -n 1
+        if [[ ${?} = 0 ]]; then
+            return 0
+        else
+            echo "waiting for the keypress"
+        fi
+    done
+}
+
+clear
+
+printf -- "We will now begin the guided pbench image test process. \n\n\n"
+
+wait_keypress 120
+
+# If somebody wants to write their own pbench-tool-meister-start, great, then
+# that person can change the assumption on where the persistent volume is for
+# the Tool Data Sink, and how the "sysinfo", "init", and "end" commands
+# operate with respect to data storage.
+
+export pbench_run=${HOME}/test-run-dir
+rm -rf ${pbench_run}
```

Comment:
As a "more formal" demo script in GitHub, it'd be better to put this under `/var/tmp` and avoid arbitrarily deleting any directory path in the user's directory.